docs/src/modules/components/Demo.js

```diff
@@ -339,7 +373,10 @@ export default function Demo(props) {
     }
   }
 
-  if (!demoOptions.demo.endsWith('.js') && demoOptions.hideToolbar !== true) {
+  if (
```

Comment:
The new demos structure uses the folder name as a prop, so we need to adjust this check.